Web/Admin.UI/src/app/components/normalization/operations/operations-sequence/operations-sequence.component.ts

```
@@ -89,10 +92,15 @@ export class OperationsSequenceComponent implements OnInit, OnDestroy {
 
   sequencesLoaded = false;
 
+  hasFacilitySequences: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)